src/assets/content/RiverConditions.js

```diff
@@ -1,14 +1,24 @@
 export default {
     riverConditionsCharts: [
+        {
+            id: '14',
+            name: 'January 1 - March 31, 2024',
+            drupal_url: 'https://www.usgs.gov/media/videos/us-river-conditions-october-december-2023',
+            folder: 'FY24_Q2/',
+            video_basename: 'river_conditions_jan_mar_2024_insta',
+            image_thumbnail: 'river_conditions_jan_mar_2024_square_thumbnail_color.png',
+            video_type: 'mp4',
+            image_alt: 'U.S. River Conditions from October 1, 2023 to December 31, 2023 at USGS streamgages.'
```

Comment:
Whoops I think your alt text needs to be switched -- this looks like it's for FY24_Q1